### DISCUSSION TITLE
Size of Play/Pause Button of current Videoplayer / Youtube Search

### DISCUSSION THREAD

**Discussion by Mumpfi3k** [FreeTubeApp/FreeTube#6422] [General]
Hi,

1.) Can you Guys please add an Option to be able to setup the Size of these Buttons or to hide them? They are way too big and they scale up with Monitor Fullscreen Resolution and they block Content.

2.) I use from time to time the Search Function to search for Content on Youtube with the same Search term. I think it would be a useful Feature when we can add Buttons to the Search Bar like a Bookmark to search with the Term.

Thx in advance.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
1. fixed in #6002 
2. Duplicate of #1505 #4040
